packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.test.tsx

```diff
@@ -202,3 +203,15 @@ describe('Render calling with respective to props', () => {
     renderMock.mockRestore();
   });
 });
+
+describe('GroupedVerticalBarChart - mouse events', () => {
+  beforeEach(sharedBeforeEach);
+  afterEach(sharedAfterEach);
+
+  it('Should render callout on hover', () => {
+    wrapper = mount(<GroupedVerticalBarChart data={chartPoints} calloutProps={{ doNotLayer: true }} />);
+    wrapper.find('rect').at(0).simulate('mouseover');
```

Comment:
are you able to simulate mouseover at different x, y coordinates as well

Reply:
I can simulate mouseover on different rect but not at different x, y coordinates as they depend on the viewport